analyzer-comments/python/two-fer/conditionals.md

```diff
@@ -0,0 +1,7 @@
+# unnecessary conditionals
+
+[Conditionals](https://github.com/exercism/python/blob/main/concepts/conditionals/introduction.md) are unnecessarily used in this solution.
+It can be improved by making use of a `default argument` in combination with either [`str.format`](https://docs.python.org/3/library/stdtypes.html#str.format) or [`f-strings`](https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals) (_otherwise known as `formatted string literals` or `interpolated strings`_).
+
+See this post on [python default arguments](https://blog.finxter.com/python-default-arguments/) for more details.  
```

Comment:
The extra spaces at the end would probably trigger format warnings when we add them.

```suggestion
See this post on [python default arguments](https://blog.finxter.com/python-default-arguments/) for more details.
```